### PR TITLE
Ioctl should allow import/export from to keystore file #1191

### PR DESCRIPTION
**Pull request by lzxm160**
work on #1191 

1、
ioctl account import keystore xx UTC--2019-05-23T09-36-39.493806681Z--3ac6ad0777e76ae6974ff89bf3a719adac600077
#xx: Enter your password of keystore, which will not be exposed on the screen.
#xx: Set password
#xx: Enter password again
New account #xx is created. Keep your password, or your will lose your private key.

2、
ioctl account import keystore xx UTC--2019-05-23T09-36-39.493806681Z--3ac6ad0777e76ae6974ff89bf3a719adac600077
#xx: Enter your password of keystore, which will not be exposed on the screen.
Error: could not decrypt key with given passphrase

3、
ioctl account import keystore xxx ./nofile
#xxx: Enter your password of keystore, which will not be exposed on the screen.
Error: keystore file "./nofile" read error

### CODE REVIEW

File: cli/ioctl/cmd/account/account.go [hunk 1]
```diff
@@ -9,12 +9,15 @@ package account
...
 	"context"
+	"crypto/ecdsa"
 	"encoding/hex"
 	"fmt"
+	"io/ioutil"
 	"strings"
 	"syscall"
 
 	"github.com/ethereum/go-ethereum/accounts/keystore"
+	ethereumcrypto "github.com/ethereum/go-ethereum/crypto"
```
Comment (zjshen14) [inline cli/ioctl/cmd/account/account.go]: ethereumcrypto -> ecrypto

File: cli/ioctl/cmd/account/accountimportkeystore.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+	"syscall"
+
+	"go.uber.org/zap"
+	"golang.org/x/crypto/ssh/terminal"
+
```
Comment (zjshen14) [inline cli/ioctl/cmd/account/accountimportkeystore.go]: no line breaker

File: cli/ioctl/cmd/account/accountimportkeystore.go [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+
+	"github.com/iotexproject/iotex-core/cli/ioctl/cmd/config"
+	"github.com/iotexproject/iotex-core/cli/ioctl/validator"
+	"github.com/iotexproject/iotex-core/pkg/log"
+)
+
+// accountImportKeyStoreCmd represents the account import keystore command
+var accountImportKeyStoreCmd = &cobra.Command{
```
Comment (zjshen14) [inline cli/ioctl/cmd/account/accountimportkeystore.go]: Can we combine the command with import/export with private key string? For example,

import/export --keystorefile (--privatekey)

File: cli/ioctl/cmd/account/accountimportkeystore.go [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+	passwordBytes, err := terminal.ReadPassword(int(syscall.Stdin))
+	if err != nil {
+		log.L().Error("failed to get password", zap.Error(err))
+		return "", err
+	}
+	password := strings.TrimSpace(string(passwordBytes))
+	for i := 0; i < len(passwordBytes); i++ {
+		passwordBytes[i] = 0
+	}
+	addr, err := newAccountByKeyStore(alias, password, args[1], config.ReadConfig.Wallet)
```
Comment (zjshen14) [inline cli/ioctl/cmd/account/accountimportkeystore.go]: Can we copy keystore file into the dir that ioctl uses by default

**Comment by lzxm160** [comment]
too many confilicts

**Pull request closed without merging**